.github/workflows/main.yml

```diff
@@ -8,6 +8,8 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+      
+      - run: rustup toolchain install stable
```

Comment:
You don't need to install Rust, it works without it [example](https://github.com/Nilstrieb/simple-std/blob/master/.github/workflows/rust.yml)

Reply:
It's because github doesn't have the newest version, that's why i did that pretty sure without that the build would fail

Reply:
I don't think t hat this will make it work well, we should probably just wait for Github to ship the new version

Reply:
Yeah it won't work with removing that